lib/packages/core/src/api/__tests__/transactionApi.spec.ts

```diff
@@ -171,7 +171,7 @@ describe('Transaction Api', () => {
             // tslint:disable:max-line-length
 
                 .onPostReply(200, mockBroadcastResponse,
-                    'relPath?requestType=sendMoneyMulti&publicKey=senderPublicKey&recipients=recipient_1:2000;recipient_2:4000&feeNQT=1000&deadline=1440')
+                    'relPath?requestType=sendMoneyMulti&publicKey=senderPublicKey&recipients=recipient_1%3A2000%3Brecipient_2%3A4000&feeNQT=1000&deadline=1440')
```

Comment:
seems odd we are encoding the `:` and not the `&`. I'm worried about double-encoding here, i.e. if someone sees this URL is not fully encoded, runs `encodeURI` against it, and gets a double-encoding bug. Would it be safer if we apply encodeURI at the end instead of this?

Reply:
Upon reading some more, this actually seems like what we want.